service/src/main/java/com/ge/predix/acs/encryption/Encryptor.java

```diff
@@ -77,6 +113,6 @@ public void setEncryptionKey(final String encryptionKey) {
         if (null == encryptionKey || encryptionKey.length() != KEY_LENGTH_IN_BYTES) {
             throw new SymmetricKeyValidationException("Encryption key must be string of length " + KEY_LENGTH_IN_BYTES);
         }
-        this.encryptionKey = encryptionKey;
+        this.secretKeySpec = new SecretKeySpec(encryptionKey.getBytes(), ALGO);
```

Comment:
Agreed. In the case the key is longer than 16 bytes, the first 16 will be used (since the `offset` is 0)